Big-Spender/readme.md

```diff
@@ -68,39 +69,64 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends
+WHERE lower(description) LIKE '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT exp.id, exp.expense_area, s.amount
+FROM expense_areas exp JOIN spends s ON (exp.id = s.expense_area_id)
+WHERE exp.expense_area = 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
+The instructions for this command were a little confusing. I was unsure if it meant the total amount for each expense_area
```

Comment:
I'd love to be able to tell you that this is a rare occurrence, but if anything a well specified report is the exception rather than the rule, especially if it comes from a non-technical user.